.storybook/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+const config = require('../config');
+const pkg = require('../package.json');
+const argv = require('yargs').argv;
+
+const globals = {
+  __DEV__: config.env === 'development',
+  __PROD__: config.env === 'production',
+  __TEST__: config.env === 'test',
+  __BASENAME__: JSON.stringify(process.env.BASENAME || ''),
+  __VERSION__: JSON.stringify(pkg.version)
+};
+const cssModulesLoader = [
+  'css-loader?sourceMap&-minimize',
+  'modules',
+  'importLoaders=1',
+  'localIdentName=[name]__[local]___[hash:base64:5]'
+].join('&');
+
+const path = require('path');
```

Comment:
Also, this seems unused

Reply:
I will make sure everything unused (old Webpack setup) will be removed in a follow-up PR.

(cfr https://github.com/teamleadercrm/teamleader-ui/pull/69#discussion_r143980511)

Reply:
Removed.